android/capacitor/src/main/java/com/getcapacitor/BridgeWebChromeClient.java

```diff
@@ -261,11 +264,11 @@ public void onGeolocationPermissionsShowPrompt(String origin, GeolocationPermiss
         // Set that we want geolocation perms for this origin
         callback.invoke(origin, true, false);
```

Comment:
This should only be called with the second param as true if the permissions are accepted, otherwise it should prompt for the permission and this shouldn't be called until the user accept (and pass true) or deny (and pass false).

Reply:
thanks nice catch. I think this was always just saying true regardless 🙃